doc/source/whatsnew/v0.24.0.txt

```diff
@@ -681,6 +681,13 @@ Other
 - :meth: `~pandas.io.formats.style.Styler.background_gradient` now takes a ``text_color_threshold`` parameter to automatically lighten the text color based on the luminance of the background color. This improves readability with dark background colors without the need to limit the background colormap range. (:issue:`21258`)
 - Require at least 0.28.2 version of ``cython`` to support read-only memoryviews (:issue:`21688`)
 - :meth: `~pandas.io.formats.style.Styler.background_gradient` now also supports tablewise application (in addition to rowwise and columnwise) with ``axis=None`` (:issue:`15204`)
+
+Regressions
+~~~~~~~~~~~
+
+- Constructing a DataFrame with an index argument that wasn't already an
+  instance of :class:`~pandas.core.Index` was broken in `4efb39f
+  <https://github.com/pandas-dev/pandas/commit/4efb39f01f5880122fa38d91e12d217ef70fad9e>`_ (:issue:`22227`).
```

Comment:
Is there a reason why we created a separate `Regressions` section in the `whatsnew` ?  Generally, we just put regression bugs in the bugs section and don't "call out" the offending commits like this.

Reply:
Works for me. 👍

Reply:
so actually 0.23.5 whatsnew is now pushed, let backport this, can you move the whatsnew. ping when pushed